### ISSUE TITLE
How to add vertical point load on a raked beam?

### ISSUE THREAD

**Issue by warem-stu** [JWock82/PyNite#108]
**How to add vertical point load on a raked beam?**
For a vertical point load on a raked beam, it is in vertical downward direction, which means in FY direction in **global** coordination system. **add_member_pt_load** only supports Fy in **local** coordination system. It is a pain if we have to multiply the vertical point load by the raked angle of beam to change it to local system. Changing Fy to FY in **add_member_pt_load** doesn't work. This is the error: **IndexError: too many indices for array: array is 1-dimensional, but 2 were indexed**. But changing Fy to FY for UDL in **add_member_dist_load** for the same situation sounds working. Do I miss something?

**Example**
For mono roof portal frame, the top rafter is a raked beam. To input the live load UDL on the top rafter, I tried **FY** instead of **Fy**, e.g., `MomentFrame.add_member_dist_load('Rafter', Direction='FY', w1=-0.65, w2=-0.65, case='LL')`, it worked. But if I wanted to input vertical point load in FY in global system on the top rafter, `MomentFrame.add_member_pt_load('Rafter', Direction='FY', P=-8.04, x=2.1, case='LL')` does not work. It will say `IndexError: too many indices for array: array is 1-dimensional, but 2 were indexed`. It is a pain if we have to input in this way `MomentFrame.add_member_pt_load('Rafter', Direction='Fy', P=-8.04*cos(alpha), x=2.1, case='LL')`.
 
![image](https://user-images.githubusercontent.com/96772792/147560535-238f8f5d-fcfb-4ca1-9941-bc7a6773a486.png)

**Update 1**
I found it would have problem if I defined more than one point load in FY in global system. Please try the followed and you will see the error.
```

   from PyNite import FEModel3D
   
   # Create a new finite element model
   MomentFrame = FEModel3D()
   
   # portal frame geometry in 'm'
   span = 7.1
   knee_height = 2.74
   
   # Add nodes
   MomentFrame.add_node('1', 0, 0, 0) # left base point
   MomentFrame.add_node('2', 0, knee_height, 0) # left knee point
   MomentFrame.add_node('3', span, knee_height+2, 0) # right knee point
   MomentFrame.add_node('4', span, 0, 0) # right base point
   
   # Define section properties in 'kN' and 'm'
   E = 205e+6 # kN/m^2
   G = 80e+6 # kN/m^2
   # 250UB31.4
   Iy_col = 4.47e-6 # m^4
   Iz_col = 44.5e-6 # m^4
   J_col = 89.3e-9 # m^4
   A_col = 4010e-6 # m^2
   # 250UC72.9
   Iy_rafter = 38.8e-6 # m^4
   Iz_rafter = 114e-6 # m^4
   J_rafter = 586e-9 # m^4
   A_rafter = 9320e-6 # m^2
   
   # Define members
   MomentFrame.add_member('Col1', '1', '2', E, G, Iy_col, Iz_col, J_col, A_col) # left portal leg
   MomentFrame.add_member('Rafter', '2', '3', E, G, Iy_rafter, Iz_rafter, J_rafter, A_rafter) # rafter
   MomentFrame.add_member('Col2', '4', '3', E, G, Iy_col, Iz_col, J_col, A_col) # right portal leg
   
   # Supports
   MomentFrame.def_support('1', support_DX=True, support_DY=True, support_DZ=True, support_RX=False, support_RY=False, support_RZ=False) # pinned
   MomentFrame.def_support('4', support_DX=True, support_DY=True, support_DZ=True, support_RX=False, support_RY=False, support_RZ=False) # pinned
   
   # DL - P1
   MomentFrame.add_member_pt_load('Rafter', Direction='FY', P=-2.36, x=2.1, case='DL')
   
   # DL - P2
   MomentFrame.add_member_pt_load('Rafter', Direction='FY', P=-12.87, x=3.5, case='DL')
   
   # Add a nodal EQ load
   # Note that the direction uses uppercase notation to indicate model global coordinate system
   MomentFrame.add_node_load('2', Direction='FX', P=11, case='Eu')
   
   # Create load combinations - LC
   LC_GRAVITY = '1.2DL+1.5LL'
   LC_DL_DW = 1.2 # DL - downward
   LC_LL = 1.5
   MomentFrame.add_load_combo(LC_GRAVITY, factors={'DL':LC_DL_DW, 'LL':LC_LL})
   
   LC_DL_LL_Eu = 'DL+0.7LL+Eu'
   MomentFrame.add_load_combo(LC_DL_LL_Eu, factors={'DL':1.0, 'Eu':1.0})
   #  MomentFrame.add_load_combo(LC_DL_LL_Eu, factors={'DL':1.0, 'LL':0.7, 'Eu':1.0})
   
   # Perform a P-Big Delta analysis on the frame
   # Note that to capture P-little delta effects the members should idealy be broken into three segments each
   MomentFrame.analyze_PDelta(log=True)
   
   # A first-order analysis could be done with the following line instead
   # MomentFrame.analyze()
   
   from PyNite import Visualization
   # characteristic Load Cases (NO Deformed shape since it is only for Load Combinations)
   Visualization.render_model(MomentFrame, text_height=0.1, deformed_shape=False, deformed_scale=50, case='DL')
   #  Visualization.render_model(MomentFrame, text_height=0.1, deformed_shape=False, deformed_scale=50, case='LL')
   Visualization.render_model(MomentFrame, text_height=0.1, deformed_shape=False, deformed_scale=50, case='Eu')
   
   # Display the deformed shape of the structure magnified 50 times with the text height 0.1 model units (m) high
   Visualization.render_model(MomentFrame, text_height=0.1, deformed_shape=True, deformed_scale=50, combo_name=LC_DL_LL_Eu)
```

**Comment by JWock82** [comment]
This appears to be a bug. I’ll try to get this fixed as soon as possible. Hopefully within the next few days.

**Comment by JWock82** [comment]
This should be fixed now as of version 0.0.54

**Comment by JWock82** [comment]
Also, the parameter `text_height` has been renamed `annotation_size`. You'll need to revise your calls to `render_model` accordingly.
